### PR TITLE
Fix the whitelist log and add ping health check

### PR DESCRIPTION
**Pull request by norbertjoni**
Whitelist log: Fixes an issue with the whitelist log generation, ensuring accurate records.

Ping health check feature: Adds a ping health check using the provided PING_URL secret for proactive monitoring.

### CODE REVIEW

File: auto_update_github_action.yml [hunk 1]
```diff
@@ -56,3 +56,11 @@ jobs:
...
         CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
         CLOUDFLARE_LIST_ITEM_LIMIT: ${{ secrets.CLOUDFLARE_LIST_ITEM_LIMIT }}
+
+    - name: Send ping request
+      if: env.PING_URL != ''
+      working-directory: cloudflare-gateway-pihole-scripts
+      env:
+        PING_URL: ${{ secrets.PING_URL }}
+      run: |
+        node -e "var https=require('https');https.get('${{ env.PING_URL }}').on('error',err=>console.log('Ping failed: '+err));"
```
Comment (mrrfv) [inline auto_update_github_action.yml:66]: Why not keep it simple and just use curl?

```suggestion
        curl "${{ env.PING_URL }}"
```

File: README.md [hunk 2]
```diff
@@ -53,6 +55,8 @@ Please note that the GitHub Action downloads the recommended blocklists and whit
 - `CLOUDFLARE_ACCOUNT_ID`: Your Cloudflare account ID
 - `CLOUDFLARE_ACCOUNT_EMAIL`: Your Cloudflare account email
 - `CLOUDFLARE_LIST_ITEM_LIMIT`: The maximum number of blocked domains allowed for your Cloudflare Zero Trust plan. Use 300000 for the free plan or if you're unsure.
+- `PING_URL`: /Optional/ Your https ping URL
```
Comment (mrrfv) [inline README.md:58]:
```suggestion
- `PING_URL`: /Optional/ The HTTP(S) URL to ping (using curl) after the GitHub Action has successfully updated your filters. Useful for monitoring.
```

**Comment by norbertjoni** [comment]
curl looks nicer :)

**Comment by mrrfv** [comment]
Thanks for the contribution!

**Pull request merged**